pkg/client/src/app/pages/identities/components/identity-form/identity-form.tsx

```diff
@@ -36,18 +36,19 @@ import {
 } from "@app/queries/identities";
 import KeyDisplayToggle from "@app/common/KeyDisplayToggle";
 
-const xmllint = require("xmllint");
 const { XMLValidator } = require("fast-xml-parser");
 export interface IdentityFormProps {
   identity?: Identity;
   onSaved: (response: AxiosResponse<Identity>) => void;
   onCancel: () => void;
+  xmlValidationFn?: (value, currentSchema) => any;
```

Comment:
Please lets avoid having technical implementation details as part of the name.
Could we please replace `xmlValidationFn` with `xmlValidator` or equivalent ?